invenio_app_ils/records/metadata_extensions.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Metadata Extensions."""
+
+from copy import deepcopy
+
+from flask import current_app
+from invenio_records_rest.schemas.fields import DateString, SanitizedUnicode
+from marshmallow import Schema
+from marshmallow.fields import Bool, Integer, List
+
+
+class MetadataExtensions(object):
```

Comment:
I guess this was a copy/paste from the other PR, not gonna check it in details...